bin/p2-rm/rm.go

```diff
@@ -147,8 +149,16 @@ func (rm *P2RM) deleteUUIDPod() error {
 	pod, err := rm.PodStore.ReadPod(rm.PodUniqueKey)
 	switch {
 	case err == nil:
-		if pod.Manifest.ID() != rm.PodID {
-			return fmt.Errorf("pod %s has a podID of %s, but %s was passed as the --pod option", rm.PodUniqueKey, pod.Manifest.ID(), rm.PodID)
+		if rm.PodID != "" {
+			if pod.Manifest.ID() != rm.PodID {
+				return fmt.Errorf("pod %s has a podID of %s, but %s was passed as the --pod option", rm.PodUniqueKey, pod.Manifest.ID(), rm.PodID)
+			}
+		} else {
+			fmt.Println(fmt.Sprintf("%s has pod ID of %s, do you wish to proceed?", rm.PodUniqueKey, pod.Manifest.ID()))
```

Comment:
yeah, i don't like needing to remember newlines but you're probably right that that's easier. wish there was a Printlnf or something. since the build is already kicked off i'm gonna leave it